riscof/main.py

```diff
@@ -140,6 +140,12 @@ def execute():
         if not os.path.exists(work_dir):
             logger.debug('Creating new work directory: ' + work_dir)
             os.mkdir(work_dir)
+        elif args.command=='run':
+            if args.testfile is None:
```

Comment:
The work directory should not be removed if the start point is from database generation or after test list generation.

Reply:
I think when you are giving database file path through --dbfile argument, removing directory cause no harm because every file present there will be rewrite anyway except may be isa_checked.yaml and platform_checked.yaml. But yes it will be better if we dont remove the directory. So I'll modify the condition.

Reply:
These files are necessary as they are used by the build stage as well as in the reports. Hence they should not be removed.

Reply:
Fixed.